src/components/myPlantDetails/imageMyPlantDetails.tsx

```diff
@@ -5,23 +5,29 @@ interface ImageMyPlantsDetailsProps {
 }
 
 const ImageMyPlantsDetails = ({ myPlant }: ImageMyPlantsDetailsProps) => {
+  const dummyImage = "https://i.ibb.co/sq4Lb8f/dummy-plant.png";
```

Comment:
Waarom is de dummyImage nu `"https://i.ibb.co/sq4Lb8f/dummy-plant.png"` en niet meer `/sad-plant.gif`?